vignettes/model_definition.Rmd

```diff
@@ -0,0 +1,129 @@
+---
+title: "Mathematical methods for `baselinenowcast`"
+description: "Description of mathematical methods used in the package"
+author: Kaitlyn Johnson
+output:
+  bookdown::html_document2:
+    fig_caption: yes
+    code_folding: show
+pkgdown:
+  as_is: true
+bibliography: library.bib
+csl: https://raw.githubusercontent.com/citation-style-language/styles/master/apa-numeric-superscript-brackets.csl
+link-citations: true
+vignette: >
+  %\VignetteIndexEntry{Mathematical methods for baselinenowcast}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+# `baselinenowcast` mathematical model
+
+The following describes the estimate of the delay distribution, the generation of the point nowcast, and the estimate of the observation error, for a partially observed or complete reporting triangle. The method assumes that the units of the delays, $d$\$ and the units of reference time $t$ are the same, e.g. weekly data and weekly releases or daily data with daily releases. This method is based on the method described by ([Wolffram et al. 2023](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1011394)) developed by the Karlsruhe Institute of Technology.
+
+### Notation
```

Comment:
before getting going with it I really like the idea of a high level overview of the general approach in a single paragraph. Perhaps with some refs to similar approaches and pointing out what it is not similar to. Perhaps this needs to go elsewhere though vs here?

Reply:
I think we could intro this with a schematic using the reporting triangle and highlighting the "blocks" being computed in the matrix? I feel like there might have been a nice slide on this in one of the talks, or can make one myself.

Reply:
Added an overview and a brief schematic diagram to visually indicate the entries being summed and compared.